bundle/package.json

```diff
@@ -88,26 +91,25 @@
 		"access": "public"
 	},
 	"dependencies": {
-		"@guardian/ab-core": "^2.0.0",
-		"@guardian/commercial-core": "^5.3.0",
+		"@guardian/ab-core": "^4.0.0",
+		"@guardian/commercial-core": "^5.4.4",
 		"@guardian/consent-management-platform": "11.0.0",
-		"@guardian/core-web-vitals": "^1.0.0",
-		"@guardian/eslint-config-typescript": "^3.0.0",
-		"@guardian/libs": "^10.0.0",
-		"@guardian/prettier": "^2.1.1",
-		"@guardian/source-foundations": "^9.0.0",
+		"@guardian/core-web-vitals": "^4.0.0",
+		"@guardian/libs": "^14.0.0",
+		"@guardian/source-foundations": "^10.0.1",
 		"@guardian/support-dotcom-components": "^1.0.7",
 		"@octokit/core": "^4.0.5",
-		"csstype": "^3.1.1",
 		"fastdom": "^1.0.11",
 		"lodash-es": "^4.17.21",
-		"madge": "^5.0.1",
 		"ophan-tracker-js": "^1.4.0",
 		"prebid.js": "guardian/prebid.js#2e3b96d",
 		"process": "^0.11.10",
 		"raven-js": "^3.27.2",
 		"tslib": "^2.4.1",
 		"web-vitals": "^2.1.2",
 		"wolfy87-eventemitter": "^5.2.9"
+	},
+	"resolutions": {
+		"typescript": "~4.9.5"
```

Comment:
Was this intentionally kept in? I think that it has prevented Typescript from actually upgrading in #1222, #899 and #881